### PR TITLE
Hotfix - Remove meilisearch

### PR DESCRIPTION
**Pull request by markmead**
Switches to the new database driver.

### CODE REVIEW

File: config/scout.php [hunk 1]
```diff
@@ -11,11 +11,11 @@
     | using Laravel Scout. This connection is used when syncing all models
     | to the search service. You should adjust this based on your needs.
     |
-    | Supported: "algolia", "meilisearch", "collection", "null"
+    | Supported: "algolia", "collection", "null"
     |
     */
 
-    'driver' => env('SCOUT_DRIVER', 'meilisearch'),
+    'driver' => env('SCOUT_DRIVER', 'mysql'),
```
Comment (alecritson) [inline config/scout.php:18]: This should be `database_index` I think?

Comment (glennjacobs) [reply config/scout.php]: Yep, @markmead can you tweak?

Comment (markmead) [reply config/scout.php]: It's already done

File: config/scout.php [hunk 2]
```diff
@@ -116,24 +116,6 @@
...
-    | See: https://docs.meilisearch.com/guides/advanced_guides/configuration.html
-    |
-    */
-
-    'meilisearch' => [
-        'host' => env('MEILISEARCH_HOST', 'http://localhost:7700'),
-        'key' => env('MEILISEARCH_KEY', null),
-    ],
-
     'mysql' => [
```
Comment (glennjacobs) [inline config/scout.php:119]: We can get rid of the mysql config also, as that's not needed.

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**